wiki/Beatmap_Editor/Song_Setup/en.md

```diff
@@ -1,222 +1,270 @@
+Set the volume of the sample set.
 
-![Detailed OD chart](img/ODTable.jpg "Detailed OD chart")
+#### Test Sample Set
 
-**Note:** The osu! timing system does not allow a [hit circle](/wiki/Hit_Objects) to be hit until the previous one has been hit or its time frame has been exceeded (resulting in a miss). With a low OD, the time frame of one circle may overlap with the next. Thus, one could hit the second note with perfect timing (after failing to hit the first note) and end up completely missing both because the time frame of the first note has not been exceeded yet.
+Click on the buttons to test out how the sample set sounds. Four buttons are provided, namely: `Normal`, `Whistle`, `Finish`, and `Clap`.
 
-### Approximate Difficulty Rating
+These sounds are additive; `Whistle` is a combination of `Normal` and `Whistle`.
 
-This is a summary of all of the settings chosen on this page. More stars mean harder maps and more [score](/wiki/Score). This is not the final star ranking of the song; it is just an approximation based on the settings you chose.
+### Misc. Toggles
 
-Audio
------
+Miscellaneous toggles.
 
-![Audio section](img/SS_Audio.jpg "Audio section")
+`Samples match playback rate (for fully-hitsounded maps)`: TODO. NO CLUE.
 
-### Default Sample Settings
+## Colours
 
-Here you can configure the hit sounds to your liking.
+![Colours section][SS_colours]
 
-If you have [timing](/wiki/Beatmap_Editor/Timing) sections (F6) that change either the hit-sound set or volume, you will not be able to adjust them here. You can click reset settings to remove them. Most mappers use timing sections to set the settings you can find here.
+Configure combo colours and playfield's background colour.
 
-#### Sample Set Selection
+### Hitcircle/Slider combos
 
-Here you can choose whether to use the Normal, Soft, or Drum built-in sample set. You can also [enable custom overrides](/wiki/Beatmap_Editor_Guides/Custom_Sample_Overrides) (Put hitsounds files inside the map's folder first).
+Configure combo colour sets.
 
-#### Sample Set Volume
+Combo colours are an important part of the beatmap's aesthetic value because there will be a lot of hit circles and sliders in the beatmap.
 
-Here you can set the volume of the sample set. This is important, because while you want to hear the hit sounds (an important aspect of osu! and osu!mania), you do not want the hit-sounds to drown out the actual song.
+Clicking a combo colour or clicking on `New Combo Colour` opens up the OS's color picker (as shown in the image, uploader used a Windows OS). Choose colors that compliment the background but the colours must not camouflage into the background.
 
-#### Test Sample Set
+Up to eight combo colors can be used, although most beatmaps use four.
 
-Click on the buttons to test out how the sample sounds. These sounds are additive, Whistle is a combination of Normal and Whistle.
+`Enable Custom Colours` checkbox: TODO. NO IDEA.
```

Comment:
If you untick this setting, the editor will just delete the `[Colours]` secion in the `.osu` file forcing the map to use the player's local skin's default clors or the default colors of the default skin. Doing this is not recommended because these colors can clash with the background of the beatmap in unfortunate cases